bower-publish.sh

```diff
@@ -15,13 +19,13 @@ rm -rf bower-chosen/*
 cp public/bower.json public/*.png public/chosen.jquery.js public/chosen.css bower-chosen/
 cd bower-chosen
 
-LATEST_VERSION=$(git diff bower.json | grep version | cut -d':' -f2 | cut -d'"' -f2 | tail -1)
+LATEST_VERSION=`git tag -l "*.*.*" | sed 's/^v//' | sort -V | tail -1`
 
-if [ -z $LATEST_VERSION ] ; then
+if [ "$LATEST_VERSION" = "$CHOSEN_VERSION" ] ; then
   echo "No Chosen version change. Skipped tagging"
 else
-  echo "Chosen version changed. Tagging version ${LATEST_VERSION}\n"
-  git tag -a "v${LATEST_VERSION}" -m "Version ${LATEST_VERSION}"
+  echo "Chosen version changed. Tagging version ${CHOSEN_VERSION}\n"
+  git tag -a "${CHOSEN_VERSION}" -m "Version ${CHOSEN_VERSION}"
```

Comment:
I believe not, this only makes sure the tag matches the main repo's tag.

But I believe #2324 is not really relevant anymore, because this script has been updated before.

Although a new release hasn't been published to the bower repo since...